src/commands/createFunction/scriptSteps/PythonScriptStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { AzureWizardPromptStep, IAzureQuickPickItem, IAzureQuickPickOptions } from '@microsoft/vscode-azext-utils';
+import { FunctionLocation, IPythonFunctionWizardContext } from './IPythonFunctionWizardContext';
+import { localize } from '../../../localize';
+
+export class PythonScriptStep extends AzureWizardPromptStep<IPythonFunctionWizardContext> {
+    public async prompt(wizardContext: IPythonFunctionWizardContext): Promise<void> {
+        const entries = await vscode.workspace.fs.readDirectory(vscode.Uri.file(wizardContext.projectPath));
```

Comment:
`AzExtFsExtra` does have a `readDirectory()` command though the return types are a little bit different.

Reply:
Noted, though I'd prefer to use the VS Code APIs when available (and reasonably convenient).  If anything, it makes it easier for the extension to support their new remote host/filesystem scenarios.  (Yes, there are lots of other things that *wouldn't* work for Functions in those scenarios, but still...)

Reply:
Well, under the hood, AzExtFsExtra is using all VS Code APIs which was the whole point of this [PR](https://github.com/microsoft/vscode-azurefunctions/pull/3268).

https://github.com/microsoft/vscode-azuretools/blob/main/utils/src/utils/AzExtFsExtra.ts#L94-L100

It just offers some QOL, like not having to wrap projectPath with `vscode.Uri.file()`, but definitely not a ship blocker or anything.

Reply:
Ah, ok, I see, then forget everything I said!  For some reason I thought `AzExtFsExtra` was still using `fs` under the hood.  :-)  Eh, in this particular case I find direct use of the VS Code API easier to read than using the wrapper.  I could go either way.